pkg/uroot/uroot.go

```diff
@@ -172,12 +172,14 @@ type Opts struct {
 	// still specify a command called init or include an /init file.
 	InitCmd string
 
-	// UinitCmd is the name of a command to link /bin/uinit to.
+	// UinitCmd is the name + args of a command to link /bin/uinit to.
```

Comment:
Can you make args a separate option?

Reply:
why?

Reply:
They'll be less mystery in how it's being parsed.

Looks like with the current implementation:
* `"uinit" "command" "hello"` => This is parsed to command `"uinit"` with args `command` and `hello`

ie, the shell-like escaping isn't applied to argv0, but is applied to the rest of the args. I'd rather this be made clearer by having argv0 be a separate option.

Reply:
good point. done!